src/Riok.Mapperly/Descriptors/Mappings/IMapping.cs

```diff
@@ -11,5 +12,7 @@ public interface IMapping
 
     ITypeSymbol TargetType { get; }
 
+    ImmutableEquatableArray<MethodParameter> Parameters { get; }
```

Comment:
`ImmutableEquatableArray<MethodParameter> Parameters { get; }` can then be moved from `IMapping` to `IUserMapping`, the parameters could be passed from `UserMethodMappingBodyBuilder.BuildMappingBody` to the context. I'd also rename to `AdditionalMappingParameters` or similar as this array does not include all parameters of the method.